Shared/Extensions/UI Extensions/UIImage+MediaItem.swift

```diff
@@ -16,6 +16,10 @@ extension UIImage: MediaItem {
         return nil
     }
 
+    var previewUrl: URL? {
+        return nil
+    }
```

Comment:
Adding this to the media item protocol. This allows for quicker preview loading.